packages/CodingStandard/src/Fixer/Property/ArrayPropertyDefaultValueFixer.php

```diff
@@ -70,8 +70,13 @@ private function isArrayPropertyDocComment(Token $token): bool
         return true;
     }
 
-    private function addDefaultValueForArrayProperty(Token $semicolonToken): void
+    private function addDefaultValueForArrayProperty(Tokens $tokens,int $semicolonPosition): void
     {
-        $semicolonToken->setContent(' = [];');
+        $tokens->insertAt($semicolonPosition, new Token(']'));
+        $tokens->insertAt($semicolonPosition, new Token('['));
+        $tokens->insertAt($semicolonPosition, new Token(' '));
```

Comment:
It's never too late to improve the code.
I don't want to replace anything in this case, just add some code.

I realize the fixer is complicated, so just [this should clear the intent](https://github.com/Symplify/Symplify/blob/2b57db63d569cf1e92b02e0943814790e88032c4/packages/CodingStandard/src/Fixer/Property/ArrayPropertyDefaultValueFixer.php#L48)